CMakeLists.txt

```diff
@@ -159,12 +159,10 @@ if(WITH_STL)
   endif()
 endif()
 
-option(WITH_OTLP "Whether to include the OpenTelemetry Protocol in the SDK" OFF)
-
-if(WITH_OTLP)
+if(DEFINED WITH_OTLP)
```

Comment:
Could we add  `option(WITH_OTLP_GRPC "Whether to include the OTLP gRPC exporter in the SDK" OFF)` here? It's related to #1926 .